### PR TITLE
Update do.m3u

### PR DESCRIPTION
**Pull request by Pecaquito**
Update do.m3u

### CODE REVIEW

File: streams/do.m3u [hunk 1]
```diff
@@ -80,19 +80,19 @@ https://cnn.essastream.com:3333/live/cromtvlive.m3u8
...
 #EXTINF:-1 tvg-id="Digital15.do",Digital 15 (480p)
 http://51.81.93.29:8084/hls/d15.m3u8
 #EXTINF:-1 tvg-id="Digital809TV.do",Digital809 TV (720p)
 https://inliveserver.com:1936/digital809tv/digital809tv/playlist.m3u8
 #EXTINF:-1 tvg-id="DigitalVision63.do",Digital Visión 63 (720p)
-https://ss3.domint.net:3120/dv6_str/digitalvision/playlist.m3u8
+https://ss5.domint.net:3024/dig_str/digitalvision63/playlist.m3u8
 #EXTINF:-1 tvg-id="DimeloTV.do",Dimelo TV (1080p)
-https://cloud2.streaminglivehd.com:19360/8062/8062.m3u8
+https://cloud2.streaminglivehd.com:19360/8000/8000.m3u8
```
Comment (PopeyeTheSai10r) [inline streams/do.m3u:91]: Works but looks very pixelated.

File: streams/do.m3u [hunk 2]
```diff
@@ -152,7 +154,7 @@ https://server2grupocam.com/megavision/MV/playlist.m3u8
 #EXTINF:-1 tvg-id="MegaVisionTV.do",Megavisión TV (480p)
 http://server3.prostudionetwork.com:1943/megavision/MV/playlist.m3u8
 #EXTINF:-1 tvg-id="Microvision10.do",Microvision 10 (720p) [Not 24/7]
-https://streaming.telecablecentral.com.do/live/MicroHD/playlist.m3u8
```
Comment (PopeyeTheSai10r) [inline streams/do.m3u:155]: This still works.

File: streams/do.m3u [hunk 3]
```diff
@@ -188,9 +190,9 @@ https://rdn.essastream.com:3544/live/puntacanatvlive.m3u8
 #EXTINF:-1 tvg-id="RETV.do",R&E TV (480p) [Not 24/7]
 https://multimedia.tmcreativos.com:1936/ekhmygctur/ekhmygctur/playlist.m3u8
 #EXTINF:-1 tvg-id="RDN.do",RDN (720p)
-https://cm.hostlagarto.com:4445/rdn/rdn.myStream/playlist.m3u8
+https://vdo2.streamgato.us:3278/live/rdntvlive.m3u8
 #EXTINF:-1 tvg-id="ReadyTVCanal6.do",Ready TV Canal 6 (720p) [Not 24/7]
-https://streaming.telecablecentral.com.do/ReadyTV/ReadyHD/playlist.m3u8
```
Comment (PopeyeTheSai10r) [inline streams/do.m3u:193]: This one is working for me.

File: streams/do.m3u [hunk 4]
```diff
@@ -152,7 +154,7 @@ https://server2grupocam.com/megavision/MV/playlist.m3u8
 #EXTINF:-1 tvg-id="MegaVisionTV.do",Megavisión TV (480p)
 http://server3.prostudionetwork.com:1943/megavision/MV/playlist.m3u8
 #EXTINF:-1 tvg-id="Microvision10.do",Microvision 10 (720p) [Not 24/7]
-https://streaming.telecablecentral.com.do/live/MicroHD/playlist.m3u8
+https://streaming.telecablecentral.com.do/live/MicroHD/manifest.m3u8
```
Comment (Alstruit) [inline streams/do.m3u:157]:
```suggestion
https://streaming.telecablecentral.com.do/live/MicroHD/playlist.m3u8
```
No need to change if it still works.

File: streams/do.m3u [hunk 5]
```diff
@@ -188,9 +190,9 @@ https://rdn.essastream.com:3544/live/puntacanatvlive.m3u8
 #EXTINF:-1 tvg-id="RETV.do",R&E TV (480p) [Not 24/7]
 https://multimedia.tmcreativos.com:1936/ekhmygctur/ekhmygctur/playlist.m3u8
 #EXTINF:-1 tvg-id="RDN.do",RDN (720p)
-https://cm.hostlagarto.com:4445/rdn/rdn.myStream/playlist.m3u8
+https://vdo2.streamgato.us:3278/live/rdntvlive.m3u8
 #EXTINF:-1 tvg-id="ReadyTVCanal6.do",Ready TV Canal 6 (720p) [Not 24/7]
-https://streaming.telecablecentral.com.do/ReadyTV/ReadyHD/playlist.m3u8
+https://streaming.telecablecentral.com.do/ReadyTV/ReadyHD/chunklist_w412740925.m3u8
```
Comment (Alstruit) [inline streams/do.m3u:195]:
```suggestion
https://streaming.telecablecentral.com.do/ReadyTV/ReadyHD/playlist.m3u8
```
No need to change if it still works.

**Review by Alstruit — APPROVED** [review]

**Review by PopeyeTheSai10r — APPROVED** [review]

**Pull request merged**